unit_test/sparse/Test_Sparse_spgemm.hpp

```diff
@@ -351,7 +378,10 @@ void test_spgemm(lno_t numRows, size_type nnz, lno_t bandwidth, lno_t row_size_v
     bool failed = false;
     int res = 0;
     try{
-      res = run_spgemm<crsMat_t, device>(input_mat, input_mat, spgemm_algorithm, output_mat);
+      if(oldInterface)
+	res = run_spgemm_old_interface<crsMat_t, device>(input_mat, input_mat, spgemm_algorithm, output_mat);
+      else
+	res = run_spgemm<crsMat_t, device>(input_mat, input_mat, spgemm_algorithm, output_mat);
```

Comment:
@seheracer some compilers will complain (with `-Werror`) about the indentation, can you add a couple spaces here so it is not flucsh with the `if` statement

Reply:
Same as above